test/apps/iodemo/io_demo.cc

```diff
@@ -124,18 +132,11 @@ template<class BufferType, bool use_offcache = false> class ObjectPool {
         }
     }
 
-    void init(size_t offcache)
-    {
-        for (size_t i = 0; i < offcache; ++i) {
-            _offcache_queue.push(get_free());
-        }
-    }
-
     inline BufferType *get()
     {
         BufferType *item = get_free();
 
-        if (use_offcache && !_offcache_queue.empty()) {
+        if (use_offcache) {
```

Comment:
now there is overhead if `use_offcache && _num_offcache == 0`

Reply:
right, use_offcache is template param. will revert.